.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: ''
+labels: 'feature request'
+assignees: ''
+
+---
+
+Have a question? Please checkout our [Slack Community](https://slack.cloudposse.com) in the `#geodesic` channel or visit our [Slack Archive](https://archive.sweetops.com/geodesic/). 
```

Comment:
```suggestion
Have a question? Please checkout our [Slack Community](https://slack.cloudposse.com) in the `#terraform` channel or visit our [Slack Archive](https://archive.sweetops.com/terraform/). 
```

Reply:
Ok I need to fix this in the template in the build harness